features/subscription_attach_restrictions.feature

```diff
@@ -0,0 +1,130 @@
+@uses.config.contract_token
+Feature: One time pro subscription related tests
+
+  Scenario Outline: Attach when onlySeries is set to none
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I set the machine token overlay to the following yaml:
+      """
+      machineTokenInfo:
+          contractInfo:
+              resourceEntitlements:
+                  - type: support
+                    affordances:
+                        onlySeries: <onlyseries>
+      """
+    When I attempt to attach `contract_token` with sudo
+    Then the machine is attached
+
+    Examples: ubuntu release
+      | release | machine_type  | onlyseries |
+      | xenial  | lxd-container | none       |
```

Comment:
Did I miss a spec? I don't think this value will ever be "none"

Reply:
I may/may not have looked at the `supportLevel` field in the onlySeries contract PR when thinking this. It will never be set to "none". :sweat: